pkg/lib/alpha/declcfg/declcfg.go

```diff
@@ -0,0 +1,109 @@
+package declcfg
```

Comment:
Pet peave/nit: What's the deal with `pkg/lib`? In Go `pkg` is sort of the convention of where to put library code in a project that _also_ has a main executable.

It has always been odd to me that we have this structure in this repo.

Reply:
I had that question as well, but sought to 'when in Rome' the problem.  
If you feel strongly, I can move all this code to pkg/alpha/fbc instead of pkg/lib/alpha/fbc.  I didn't have enough historical context for a good position.

Reply:
Moved.